portal-impl/src/META-INF/portal-model-hints.xml

```diff
@@ -629,7 +629,9 @@
 		<field name="userName" type="String" />
 		<field name="createDate" type="Date" />
 		<field name="modifiedDate" type="Date" />
-		<field localized="true" name="name" type="String" />
+		<field localized="true" name="name" type="String">
+			<validator name="required" />
+		</field>
 		<field name="billingAllowed" type="boolean" />
 		<field name="shippingAllowed" type="boolean" />
 		<field name="a2" type="String" />
```

Comment:
We need to add `<validator name="number" />` for the position field. See commerce's [portlet-model-hints.xml](https://github.com/liferay/liferay-portal/blob/master/modules/apps/commerce/commerce-service/src/main/resources/META-INF/portlet-model-hints.xml#L94-L96)